packages/cli/src/platform/$.ts

```diff
@@ -12,9 +13,7 @@ export class $ implements Command {
   public async parse(argv: string[]) {
     const isHasEarlyAccessFeatureFlag = Boolean(argv.find((_) => _.match(/early-access-feature/)))
     if (!isHasEarlyAccessFeatureFlag) throw new EarlyAccessFlagError()
-
-    await Promise.resolve('todo')
-
-    return ''
+    const result = await dispatchToSubCommand(this.commands, argv)
+    return JSON.stringify(result)
```

Comment:
A nitty one:

```suggestion
return result
```,

or could be:
```suggestion
return await dispatchToSubCommand(this.commands, argv)
```,

Reply:
Not sure I understand, removing the JSON stringification breaks the CLI output. This result goes direct to `console.log` so it cannot be the object since under the hood that runs `util.format` etc. which is not built for JSON.parse consumption that we need on the output (e.g. for `| jq`).

Reply:
The proposal was inspired from command parsing done by other commands. For example `MigrateCommand`

In this example, I like that the `unknownCommand` utility is used together with the `help` property of the class

https://github.com/prisma/prisma/blob/91cb420813b559bcf941b535f1cc606a23c0838c/packages/migrate/src/commands/MigrateCommand.ts#L85-L99

Reply:
> We'd not doing help right now though.

Even if not, for now we could have an empty help block which we could then plug in to already existing pattern. Later on, we can add the help descriptions.

Reply:
But we don't know if we're going to be using that pattern yet.